kythe/cxx/indexer/textproto/plugin.h

```diff
@@ -94,6 +94,15 @@ class Plugin {
       const google::protobuf::FieldDescriptor& field,
       const std::vector<StringToken>& tokens) = 0;
 
+  // Optional entrypoint for integer fields. Plugin may override it to add
+  // additional nodes for integer fields.
+  virtual absl::Status AnalyzeIntegerField(PluginApi* api,
+                                           const proto::VName& file_vname,
+                                           const proto2::FieldDescriptor& field,
```

Comment:
externally these are in the `google::protobuf` namespace

Reply:
Thanks! Missed it. Updated.

https://buildkite.com/bazel/kythe/builds/6875